src/Connection.php

```diff
@@ -383,8 +388,8 @@ private function getCriteriaCondition(array $criteria): array
      *
      * Table expression and columns are not escaped and are not safe for user-input.
      *
-     * @param array<string, mixed>                                                                  $criteria
-     * @param array<int<0,max>, string|ParameterType|Type>|array<string, string|ParameterType|Type> $types
+     * @param array<string, mixed>                                                        $criteria
+     * @param array<int<0,max>, WrapperParameterType>|array<string, WrapperParameterType> $types
```

Comment:
Technically, this is a breaking changes for extending classes, however I think it's not a big deal.

Reply:
It should be documented though.